packages/translations/src/commonMessages.ts

```diff
@@ -227,6 +232,11 @@ export const commonMessages = {
     message: 'Document signed',
     comment: 'Shown when signing a document succeeded',
   }),
+  signingPending: defineMessage({
```

Comment:
Signing is during presentation, which doesn't support deferred issuance, so not sure when this text is used?